requirements_test.txt

```diff
@@ -2,12 +2,16 @@
 # make new things fail. Manually update these pins when pulling in a
 # new version
 
--r requirements_test_pre_commit.txt
+# When updating this file, update .pre-commit-config*.yaml too
```

Comment:
This file shouldn't change in this PR. Please rebase on latest dev branch.

Reply:
I overwrite requirements_test.txt with latest dev version. Is it correct or I must follow a different procedure?

Reply:
If it doesn't change compared to dev branch it's good.